ci/ray_ci/runner.py

```diff
@@ -72,9 +87,6 @@ def _get_test_targets(
     """
     Get test targets to run for a particular shard
     """
-    if not yaml_dir:
-        yaml_dir = os.path.join(bazel_workspace_dir, "ci/ray_ci")
-
```

Comment:
why are these removed? I thought you say this is the right place?

now it seems that it will be executed for multiple times?

Reply:
then maybe remove the `Optional` and default `None` value in the arg list.